base/docs/helpdb/Base.jl

```diff
@@ -2082,7 +2082,11 @@ kron
 """
     >>(x, n)
 
-Right bit shift operator, preserving the sign of `x`.
+Signed right bit shift operator, `x >> n`. The result is `x` shifted right by `n` bits,
```

Comment:
Good point: I hadn't noticed that the code in `intrinsics.cpp` used an unsigned comparison. However, that's not quite true: `big(1) << -1` returns an error, not `big(1) << unsigned(-1)` (which would be huge).